chaos.proto

```diff
@@ -119,6 +119,23 @@ message LineSection{
     repeated PtObject routes = 5;
 }
 
+message OrderedPtObject{
+ required PtObject pt_object = 1;
+ required uint32 order = 2;
+}
+
+message RoutePattern{
+ repeated OrderedPtObject ordered_pt_objects = 1;
+}
+
+message RailSection {
+ required PtObject line = 1;
```

Comment:
required or optional ?

Reply:
For the moment we agreed to leave it mandatory.

Reply:
This is not required anymore, we should change it.